From 7513aa3fce5758dd54b8a6729083f9be4f76374f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Sat, 10 Mar 2018 11:22:26 -0800
Subject: [PATCH] TRITON-226 TRITON-137 broke sapi proto mode setup (i.e.
 headnode setup) Reviewed by: Dylan Yep <dyep49@gmail.com> Approved by: Dylan
 Yep <dyep49@gmail.com>

---
 boot/setup.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/boot/setup.sh b/boot/setup.sh
index 007c577..6222817 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -66,6 +66,7 @@ MORAY_ADMIN_IPS=${moray_admin_ips}
 NAPI_ADMIN_IPS=${napi_admin_ips}
 VMAPI_ADMIN_IPS=${vmapi_admin_ips}
 CNAPI_ADMIN_IPS=${cnapi_admin_ips}
+SAPI_ADMIN_IP=${sapi_admin_ips}
 EOF
     )
 
@@ -88,6 +89,10 @@ EOF
     "level": "debug"
   },
   "datacenter_name": "$DATACENTER_NAME",
+  "serviceName": "{{SERVICE_NAME}}",
+  "instanceUuid": "{{auto.ZONENAME}}",
+  "serverUuid": "{{auto.SERVER_UUID}}",
+  "adminIp": "$SAPI_ADMIN_IP",
   "moray": {
     "host": "$MORAY_HOST",
     "port": 2020
-- 
2.21.0

